test/libethereum/BlockchainTestsFiller/TestNetwork/bcEIP150TestFiller.json

```diff
@@ -0,0 +1,335 @@
+            "coinbase" : "0x8888f1f195afa192cfee860698584c030f4c9db1",
+            "difficulty" : "131072",
+            "extraData" : "0x42",
+            "gasLimit" : "31415920",
+            "gasUsed" : "0",
+            "mixHash" : "0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421",
+            "nonce" : "0x0102030405060708",
+            "number" : "0",
+            "parentHash" : "0x0000000000000000000000000000000000000000000000000000000000000000",
+            "receiptTrie" : "0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421",
+            "stateRoot" : "0xf99eb1626cfa6db435c0836235942d7ccaa935f1ae247d3f1c21e495685f903a",
+            "timestamp" : "0x54c98c81",
+            "transactionsTrie" : "0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421",
+            "uncleHash" : "0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347"
+        },
+        "expect" : {
+            "1000000000000000000000000000000000000001" : {
+                "storage" : {
+                    "0x00" : "0x12",
+                    "0x01" : "0x20",
+                    "0x02" : "0x1122334455667788991011121314151617181920000000000000000000000000",
+                    "0x03" : "0x17486dc036",
+                    "0x04" : "0x12",
+                    "0x05" : "0x01",
+                    "0x06" : "0x01",
+                    "0x07" : "0x01",
+                    "0x08" : "0x01",
+                    "0x09" : "0x057ee9",
+                    "0x64" : "0x11"
+                }
+	     },
+            "1000000000000000000000000000000000000101" : {
+                "storage" : {
+                    "0x00" : "0x12",
+                    "0x01" : "0x20",
+                    "0x02" : "0x1122334455667788991011121314151617181920000000000000000000000000",
+                    "0x03" : "0x17486a26f2",
+                    "0x04" : "0x12",
+                    "0x05" : "0x01",
+                    "0x06" : "0x01",
+                    "0x07" : "0x01",
+                    "0x08" : "0x01",
+                    "0x09" : "0x059447",
+                    "0x64" : "0x11"
+                }
+            },
+	    "1000000000000000000000000000000000000002" : {
+                "balance" : "0x2710"
+	    },
+	    "1000000000000000000000000000000000000012" : {
+		"shoulsnotexist" : "1"
+	    },
+	    "1000000000000000000000000000000000000013" : {
+		"shoulsnotexist" : "1"
+	    }
+        },
+        "pre" : {
+            "d4fe7bc31cedb7bfb8a345f31e668033056b2728" : { "balance" : "1000000", "nonce" : "0", "code" : "", "storage": {} },
```

Comment:
Is this the attack contract? If yes, I don't think this is a good test. What we need to test here is gas prices, not resource usage.